### PR TITLE
neaten up/tabsCheckout

### PR DESCRIPTION
**Pull request by Jood80**
### modification 

- [x] make copyright function as a footer
- [x] split the confirmation tab from the TabCheckout shared component 
I've initiated each folder with a **xxx.jsx** file named with the component functionality and another **index.js** file
please let me know if we can change our file structure into this type or just staying at the one that we're using currently

### Testing
1. write these following commands down on your terminal :
```
$ pull origin neatingUp/tabsCheckout
$ npm run client
```
2. browse http://localhost:3000/reservation.

### CODE REVIEW

File: client/src/components/layouts/ConfirmationTab/confirmationTab.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { Fragment } from 'react';
+import { Typography } from '@material-ui/core';
+
+const Confirmation = () => (
+  <Fragment>
+    <Typography variant="h6" gutterBottom>
+      Thank you!
+    </Typography>
+    <Typography variant="h6" gutterBottom>
+      Our health and saftey rules
```
Comment (okjake) [inline client/src/components/layouts/ConfirmationTab/confirmationTab.jsx]: "safety"

File: client/src/components/layouts/ConfirmationTab/confirmationTab.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    </Typography>
+    <Typography variant="subtitle1">
+      1. All visitors must wear face masks before entering the building.
+      <br />
+      2. No more than 3 People allowed to be in the showroom at once.
+      <br />
+      3. Make sure to follow the American Red Cross Instuctions of Saftey.
```
Comment (okjake) [inline client/src/components/layouts/ConfirmationTab/confirmationTab.jsx]: Safety

**Comment by okjake** [comment]
"_I've initiated each folder with a xxx.jsx file named with the component functionality and another index.js file
please let me know if we can change our file structure into this type or just staying at the one that we're using currently_"  I think this is fine to do, it makes your imports more concise and readable

**Review by okjake — CHANGES_REQUESTED** [review]
This looks good to me, there's just a couple of spelling mistakes with 'safety' - I saw that this came from the design work so we should consider it for the rest of the app too

We might want to do some small styling tweaks but I'd suggest we leave those to the end of the project and do them in one go after all the functionality is in

**Comment by Jood80** [comment]
> This looks good to me, there's just a couple of spelling mistakes with 'safety' - I saw that this came from the design work so we should consider it for the rest of the app too
> 
> We might want to do some small styling tweaks but I'd suggest we leave those to the end of the project and do them in one go after all the functionality is in

Alright :blush:

**Review by sara219 — APPROVED** [review]

**Review by tasneembhiri — APPROVED** [review]

**Review by Rahaf-96 — APPROVED** [review]

**Pull request merged**